bigbluebutton-html5/imports/api/2.0/cursor/server/handlers/cursorUpdate.js

```diff
@@ -0,0 +1,16 @@
+import { check } from 'meteor/check';
+import updateCursor from '../modifiers/updateCursor';
+
+export default function handleCursorUpdate({ header, body }) {  
+  const meetingId = header.meetingId;
```

Comment:
We are unpacking in every handler the meetingId from the header. Inst easier to abstract this somewhere? Maybe on the RedisPubSub class? Same for userId and the reply_to stuff.